GeoDaCenter/chicago-environment-explorer
# feat: adjust index builder metadata names and values

## Problem
A few new columns have been added to the Google sheet to support the Index Builder

Fixes #136 

## Approach
* feat: added `ibEnabled` / `ibIgnoreZero` / `ibDefaultDirectionality` / `ibExplanation` in source data sheet (with comments for descriptions of each new column)
* feat: adjusted code references: `scaling` -> `ibDefaultDirectionality`, `ignore_0` -> `ibIgnoreZero`
* feat: added support for `ibEnabled` - this flag will hide indicators ONLY on the Index Builder
* feat: added new column `ibExplanation` to data sheet - this is not yet used by the UI, but we could use this to provide variable-specific context to the user in the UI
* fix: remove original source data for `scaling` / `ignore_0` from data sheet
* fix: set WeightsSliders so that it does not appear below the HelperText - it should now only appear to the right

## How to Test
1. Navigate to /builder
    * You should see that "Displacement Pressure" and other variables where `ibEnabled=FALSE` do not appear in this list
2. Select 1 or more indicators and click Next
    * You should be brought to the "Choose Weights" page
    * You should see that the sliders only appear to the right, and not below the HelperText
3. Zoom in a few times with your browser (Cmd + for Mac, Ctrl + for Windows)
    * You should see that the sliders only appear to the right, and not below the HelperText
4. Select weights using the sliders and click Next again 
5. Verify that values for `ibDefaultDirectionality` properly affect the mapped index data
